subprojects/android-test/ui-testing-core-app/src/main/res/values/strings.xml

```diff
@@ -1 +1,4 @@
-<resources></resources>
+<resources>
+
+    <string name="test_string_1">test string we expect</string>
```

Comment:
Let's try to make more meaningful name and content.
E.g. 
```xml
<string name ="string_with_non_breaking_spaces">string with non breaking spaces</string>
```

Reply:
Updated